### PR TITLE
this works?

### PR DESCRIPTION
**Pull request by juliusmarminge**

### CODE REVIEW

File: packages/nextjs/package.json [hunk 1]
```diff
@@ -2,41 +2,26 @@
...
-  ],
+  "keywords": ["create-t3-app", "environment variables", "zod", "nextjs"],
   "author": "Julius Marminge",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/t3-oss/t3-env",
     "directory": "packages/nextjs"
   },
-  "main": "dist/index.js",
```
Comment (fysp11) [inline packages/nextjs/package.json:18]: I believe this line is breaking the import. Typescript just can't find it anymore. It works on 0.9.1

Comment (fysp11) [reply packages/nextjs/package.json:18]: Any chance this was an wrong change, @juliusmarminge ?

Comment (juliusmarminge) [reply packages/nextjs/package.json:18]: It's only breaking if your tsconfig cant read exports field

Comment (A7med3bdulBaset) [reply packages/nextjs/package.json:18]: same to me: 

My tsconfig:

```json
{
  "compilerOptions": {
    "tsBuildInfoFile": "node_modules/.cache/typescript/tsbuildinfo.json",
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "target": "ES2022",
    "esModuleInterop": true,
    "module": "ESNext",
    "moduleResolution": "Node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "lib": ["ESNext", "DOM", "DOM.Iterable"],
    "noUncheckedIndexedAccess": true,
      "incremental": true,
    "baseUrl": ".",
    "paths": {
      "@/*": ["./*"]
    },
    "plugins": [
      {
        "name": "next"
      }
    ],
  },
  "include": [
    "next-env.d.ts",
    "next-auth.d.ts",
    "**/*.ts",
    "**/*.tsx",
    ".next/types/**/*.ts"
  ],
  "exclude": ["node_modules"]
}
```

Comment (A7med3bdulBaset) [reply packages/nextjs/package.json:18]: I had to change 
```json
    "module": "ESNext",
    "moduleResolution": "Node",
```
to
```json
    "module": "NodeNext",
    "moduleResolution": "NodeNext",
```

and it's working now

Comment (nbibler) [reply packages/nextjs/package.json:18]: I'm not seeing typings being locatable or available in `0.10.1` when installed. Additionally, npmjs.org doesn't report them as being available, either. See the screenshot of released versions:

npmjs-typescript

Comment (juliusmarminge) [reply packages/nextjs/package.json:18]: You'll get types through exports condition if you have a modern moduleResolution

Comment (nbibler) [reply packages/nextjs/package.json:18]: @juliusmarminge : Ahh ... you're correct. Using something like `moduleResolution: "bundler"` correctly identifies the types. Thank you!

**Pull request merged**